server/src/main/java/org/elasticsearch/index/mapper/CompletionFieldMapper.java

```diff
@@ -435,9 +435,14 @@ public Mapper parse(ParseContext context) throws IOException {
         XContentParser parser = context.parser();
         Token token = parser.currentToken();
         Map<String, CompletionInputMetaData> inputMap = new HashMap<>(1);
+
+        // ignore null values
         if (token == Token.VALUE_NULL) {
-            throw new MapperParsingException("completion field [" + fieldType().name() + "] does not support null values");
-        } else if (token == Token.START_ARRAY) {
+            context.addIgnoredField(fieldType.name());
```

Comment:
I don't think we need to add explicit `null` values to the `_ignored` field. It should be treated as if the field was missing.

Reply:
stopped adding the field to the ignored list